src/ng1/directives/floatingActionButton/floatingActionButton.controller.js

```diff
@@ -1,9 +1,45 @@
-FloatingActionButton.$inject = ["$scope"];
+export default class FloatingActionButton {
 
-export default function FloatingActionButton() {
-	var vm = this;
+	constructor($scope, $document, $element) {
```

Comment:
Can you check the indentation in this file and ensure that it is consistent with the rest of the project? (Spaces instead of tabs.)